### DISCUSSION TITLE
Unable to authenticate to foundryvtt.com on QNAP: getaddrinfo EAI_AGAIN

### DISCUSSION THREAD

**Discussion by kenosama** [felddy/foundryvtt-docker#200] [Q&A]
I'm struggling with th installation on a qnap nas via container station, it always send an error.
the model is a TS 453 A 4G, on the latest version of QTS and Container station. 

## To Reproduce

i'm using theses setting to setup the server
![image](https://user-images.githubusercontent.com/38300951/119737235-3daffd80-be7f-11eb-8938-b2fe39d4c170.png)

## Expected behavior

Normally it would fetch the latest version of FVTT install it and let me setup the program but i get those logs here under

## Any helpful log output

```console
Entrypoint | 2021-05-26 22:06:20 | [info] Starting felddy/foundryvtt container v0.7.9                                                          
Entrypoint | 2021-05-26 22:06:20 | [info] No Foundry Virtual Tabletop installation detected.                                                   
Entrypoint | 2021-05-26 22:06:20 | [info] Using FOUNDRY_USERNAME and FOUNDRY_PASSWORD to authenticate.                                         
Authenticate | 2021-05-26 22:06:22 | [info] Requesting CSRF tokens from https://foundryvtt.com                                                 
Authenticate | 2021-05-26 22:06:28 | [error] Unable to authenticate: request to https://foundryvtt.com/ failed, reason: getaddrinfo EAI_AGAIN f oundryvtt.com     
```             

i'm really not a technical guy and i don't know how to setup this docker image correctly. 
if you have any idea i'm eager to help you as much as i can.

**Comment by felddy** [comment]
The `getaddrinfo EAI_AGAIN` error means that your NAS either has an internet connectivity issue or a bad DNS resolver configuration.   That is, the container is unable to resolve the IP address for `foundryvtt.com`.  

Make sure the DNS entries in the "Preferences" tab of the "Container Station" are correct.  You usually do not need to change these values. 

Also, you can try executing `nslookup foundryvtt.com` in the container to see if it can resolve the DNS entry.

**Reply by InTheSpaghetti** [reply]
I am getting the same  getaddrinfo EAI_AGAIN error. 
I don't see the "Preferences" tab of the "Container Station" to ensure the DNS entries are correct. Has this tab moved after an update? I did not change them. I also cannot test using nslookup as the container stops itself as soon as the error occurs.

I'd really like to get this resolved. We haven't had a session since January because the GM moved and has a poor internet connection which is why I'm trying to host from my Synology NAS using Docker.

Any help is appreciated and I will provide more info if needed.

**Reply by madereddy** [reply]
@InTheSpaghetti Still need help? I have foundry running on Synology.
